tests/interop/test_calls.py

```diff
@@ -61,7 +60,7 @@ async def __aexit__(self, *tb):
 class TestCalls(aiotest.TestCase):
     async def call_t_a(self, proc, *args, loop=None):
         """called from Trio"""
-        return await aio_as_trio(proc, loop=loop)(*args)
```

Comment:
I don't think we should stop passing loop= here. The loop for functions like `asyncio.sleep` is obvious (the running loop) but in this case we might be scheduling work on a loop that isn't the ambient one.

Reply:
I think this was motivated by the following deprecation warning:

> .../trio-asyncio/tests/test_misc.py:266: DeprecationWarning: The loop argument is deprecated since Python 3.8, and scheduled for removal in Python 3.10.